### DISCUSSION TITLE
How to use AdonisJS v6 Repl to read data

### DISCUSSION THREAD

**Discussion by wswawan** [adonisjs/core#4455] [Help] [answered]
In another version adonisjs to read data in Repl I can use query() why in v6 I can't use the query builder like below this

```
await loadModels()
await models.user.query().select()
```

but for create method it works

with adonisjs v6 Repl to read data I want to be able to use query builders like all(), find(), findBy() and others

**Comment by Julien-R44** [comment]
not sure if i understand your problem. the REPL code you just posted works fine. what's the error or issue you are facing?

```ts
await loadModels()
const user = await models.user.query().select().first()
user.name // will print the user name in the repl
```

**Reply by wswawan** [reply]
I want to read all values in table users, like using sql query `SELECT * FROM users;` or using limit `SELECT * FROM users limit 2;`, `first()` method works fine.

**Reply by Julien-R44** [reply]
well, can you show the code that is not working? because, this will be the same thing : 

```ts
await loadModels()
const users = await models.user.query().select()
const usersLimit = await models.user.query().select().limit(2)
```

i think, you are probably missing an `await` before your call

**Reply by wswawan** [reply]
this is the capture of my results

<img width="966" alt="Screenshot 2024-03-12 at 9 47 28 AM" src="https://github.com/adonisjs/core/assets/13808745/8327ed4c-fe9a-4bf3-9da1-fec3aa7b3294">

if using variable `db` from `await loadDb()` it works fine

<img width="966" alt="Screenshot 2024-03-12 at 10 14 04 AM" src="https://github.com/adonisjs/core/assets/13808745/888ce4d9-91c1-4716-b988-5b5639f084a1">

**Reply by Julien-R44** [reply, marked as answer]
yeah that's normal. in the first case, it returns instances of `User`. in the second case, it's just standard json, plain POJOs

read [this](https://lucid.adonisjs.com/docs/models#what-is-the-active-record-pattern)

in your first screenshot, if you write `users[0].id` you'll get the id displayed in the REPL

**Comment by wswawan** [comment]
thank you
